src/pymovements/dataset/dataset_download.py

```diff
@@ -34,6 +34,7 @@ def download_dataset(
         paths: DatasetPaths,
         extract: bool = True,
```

Comment:
can you add a `*,` before `extract`? This is PR a breaking change either way, so we can make it more future proof.